docs/user/versions.rst

```diff
@@ -160,3 +92,25 @@ for example ``https://pip.readthedocs.io/``,
 they will be redirected to the **Default version**.
 This defaults to **latest**,
 but could also point to your latest released version.
+
+Versioning workflows
+--------------------
+
+Read the Docs makes certain assumptions about your documentation version defaults,
+all of which can be reconfigured if necessary:
+
+- ``latest`` version points to the most up to date development code.
+  If you develop on a branch that is different than the default for your version control system,
+  set the **Default Branch** to the branch you use.
+
+- **tags** are semantic versioning compatible (according to  `PEP 440`_) snapshots
+  of your documentation. The most recent semantic tag maps to the ``stable`` tag.
```

Comment:
> of your documentation. The most recent semantic tag maps to the ``stable`` tag.

I think this should say "of your documentation. The most recent semantic tag maps to the ``stable`` version". Right?

Reply:
yep, fix incoming.

Reply:
while we're here. What is more accurate than "most recent semantic tag"? Something that implies the highest stage version number, not the most recent in time?

Reply:
Maybe "the highest semantic version", 🤔 ? I'm not really good at English 😅